cmd/collector/app/server/zipkin.go

```diff
@@ -39,8 +39,7 @@ type ZipkinServerParams struct {
 	TLSConfig      tlscfg.Options
 	HostPort       string
 	Handler        handler.ZipkinSpansHandler
-	AllowedOrigins string
-	AllowedHeaders string
+	CORSSettings   corscfg.Options
```

Comment:
to keep the existing naming conventions:
```suggestion
	CORSConfig   corscfg.Options
```